src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -20,60 +20,59 @@
-    {@processResponse(sample)}
-  @case "RequestPaged"
-    {@pagedOptionalArrayMethodSampleCodePaged(apiMethod, sample)}
-  @case "RequestPagedAll"
-    {@pagedOptionalArrayMethodSampleCodeAll(apiMethod, sample)}
-  @case "RequestStreamingBidi"
-    {@optionalArrayMethodSampleCodeBidiStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "RequestStreamingClient"
-    {@optionalArrayMethodSampleCodeClientStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "RequestStreamingServer"
-    {@optionalArrayMethodSampleCodeServerStreaming(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @case "LongRunningPromise"
-    {@lroSampleCode(apiMethod, sample.initCode)}
-    {@processResponse(sample)}
-  @default
-    $unhandledCallingForm: {@sample.callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
-  @end
-  
-  {@standaloneSampleTeardown(apiMethod, sample)}
-@end
-
-@private standaloneSampleSetup(apiMethod, sample)
   @# [START {@sample.regionTag}]
-  
+
   {@importList(sample.initCode.importSection.appImports)}
 
-  @# [START {@sample.regionTag}_core]
-  
-  client = {@apiMethod.apiModuleName}.{@apiMethod.apiClassName}()
+  def some_func({@sampleFuncParam(sample.initCode.argDefaultParams)}):
+    @# [START {@sample.regionTag}_core]
+
+    client = {@apiMethod.apiModuleName}.{@apiMethod.apiClassName}()
+
+    @if sample.initCode.argDefaultLines
+      @join line : util.pretty(initCode(sample.initCode.argDefaultLines))
+        @# {@line}
+      @end
+
 
-  @if sample.initCode.argDefaultLines
-    @join line : util.pretty(initCode(sample.initCode.argDefaultLines))
-      @# {@line}
     @end
+    @if sample.initCode.lines
+      {@initCode(sample.initCode.lines)}
 
+    @end
+    @switch sample.callingForm
+    @case "Request"
+      {@optionalArrayMethodSampleCodeNonStreaming(apiMethod, sample.initCode)}
+      {@processResponse(sample)}
+    @case "RequestPaged"
+      {@pagedOptionalArrayMethodSampleCodePaged(apiMethod, sample)}
+    @case "RequestPagedAll"
+      {@pagedOptionalArrayMethodSampleCodeAll(apiMethod, sample)}
+    @case "RequestStreamingBidi"
+      {@optionalArrayMethodSampleCodeBidiStreaming(apiMethod, sample.initCode)}
+      {@processResponse(sample)}
+    @case "RequestStreamingClient"
+      {@optionalArrayMethodSampleCodeClientStreaming(apiMethod, sample.initCode)}
+      {@processResponse(sample)}
+    @case "RequestStreamingServer"
+      {@optionalArrayMethodSampleCodeServerStreaming(apiMethod, sample.initCode)}
+      {@processResponse(sample)}
+    @case "LongRunningPromise"
+      {@lroSampleCode(apiMethod, sample.initCode)}
+      {@processResponse(sample)}
+    @default
+      $unhandledCallingForm: {@sample.callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
+    @end
 
-  @end
-  @if sample.initCode.lines
-    {@initCode(sample.initCode.lines)}
+    @# [END {@sample.regionTag}_core]
 
-  @end
+  @# [END {@sample.regionTag}]
```

Comment:
We need to include a main that will call the sample function with the default values (for now; later it will process any configured flags)

The main would exist after the `[END {@sample.regionTag}]`

Reply:
After today's discussion, the main func should just pass CLI args after type conversion right? Let's do that separately?

Reply:
Done